src/context/index.ts

```diff
@@ -64,6 +64,7 @@ export const setupContext = async (config: Config): Promise<DirectoryContext | Y
   })(config);
 
   const accessToken = await (async (): Promise<string> => {
+    if (config.AUTH0_DOMAIN === 'deploy-cli-dev.eu.auth0.com') return '[REDACTED]';
```

Comment:
This hacky bit is required because we're redacting the access token when we post-process the Nock recordings. However, Nock will try to proceed through the normal authentication flow and be unable to match the non-redacted authentication requests with the redacted recordings. By adding this line of code here, we're immediately bypassing the access token generation because it's not needed by the recordings.